docker/dockerfiles/Dockerfile.onnx.cpu.dev

```diff
@@ -0,0 +1,78 @@
+FROM python:3.9 as base
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt update -y && apt install -y \
+    libxext6 \
+    libopencv-dev \
+    uvicorn \
+    python3-pip \
+    git \
+    libgdal-dev \
+    cmake \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY requirements/requirements.sam.txt \
+    requirements/requirements.clip.txt \
+    requirements/requirements.cpu.txt \
+    requirements/requirements.vino.txt \
+    requirements/requirements.http.txt \
+    requirements/requirements.waf.txt \
+    requirements/requirements.gaze.txt \
+    requirements/requirements.doctr.txt \
+    requirements/requirements.groundingdino.txt \
+    requirements/requirements.yolo_world.txt \
+    requirements/requirements.transformers.txt \
+    requirements/_requirements.txt \
+    ./
+
+RUN pip3 install --upgrade pip  && pip3 install \
+    -r _requirements.txt \
+    -r requirements.sam.txt \
+    -r requirements.clip.txt \
+    -r requirements.cpu.txt \
+    -r requirements.http.txt \
+    -r requirements.waf.txt \
+    -r requirements.gaze.txt \
+    -r requirements.doctr.txt \
+    -r requirements.groundingdino.txt \
+    -r requirements.yolo_world.txt \
+    -r requirements.transformers.txt \
+    jupyterlab \
+    wheel>=0.38.0 \
+    --upgrade \
+    && rm -rf ~/.cache/pip
```

Comment:
Caching can be turned off by adding `--no-cache-dir` to pip (so if pip changes their cache dir in the future we won't have to update this Dockerfile)